docs/design/lcevc-integration.md

```diff
@@ -0,0 +1,266 @@
+
+# Shaka Player LCEVC integration
+
+# Introduction
+
+This article describes the V-Nova LCEVC Shaka Player integration.
+
+Shaka Player is an open-source JavaScript library for adaptive media. It plays adaptive media formats (such as [DASH](http://dashif.org/) and [HLS](https://developer.apple.com/streaming/)) in a browser, without using plugins or Flash. Instead, Shaka Player uses the open web standards [MediaSource Extensions](https://www.w3.org/TR/media-source/) and [Encrypted Media Extensions](https://www.w3.org/TR/encrypted-media/).
+
+The official project GitHub repository is <https://github.com/google/shaka-player>. This integration is a fork from the Shaka Player open source project from version 3.3.0-pre with commit hash: 73b430248ba76038b466da902b776ecc920b2a35.
```

Comment:
I think these lines can be dropped for this PR to be merged into the repo.

Reply:
These lines are deleted.